presto-tablestore/src/main/java/com/facebook/presto/tablestore/RowSearchIterator4CountStar.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tablestore;
+
+import com.alicloud.openservices.tablestore.model.Row;
+import com.alicloud.openservices.tablestore.model.iterator.SearchRowIterator;
+
+import java.util.Iterator;
+
+public class RowSearchIterator4CountStar
+        implements Iterator<Row>
+{
+    private final SearchRowIterator inner;
+    private final long totalRow;
+    private long row;
+
+    public RowSearchIterator4CountStar(SearchRowIterator inner)
+    {
+        this.inner = inner;
+        this.totalRow = inner.getTotalCount();
+    }
+
+    @Override
+    public boolean hasNext()
+    {
+        return row++ < totalRow;
+    }
+
+    @Override
+    public Row next()
+    {
+        return null;
```

Comment:
Is returning null an expected behavior here?

Reply:
ping